app/components/support_form_component.html.erb

```diff
@@ -55,15 +54,27 @@
             </dd>
           </div>
         <% end %>
+
+        <% if cohort_year.present? %>
+          <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">
+              Cohort
+            </dt>
+            <dd class="govuk-summary-list__value">
+              <%= cohort_year %>
```

Comment:
Can this output the academic year (ie "2023 to 2024") to match the previous page, rather than just a single year, which is ambiguous?

Reply:
Updated.